src/main/java/io/reactivex/Scheduler.java

```diff
@@ -13,17 +13,20 @@
 
 package io.reactivex;
 
-import java.util.concurrent.TimeUnit;
-
-import io.reactivex.annotations.*;
+import io.reactivex.annotations.NonNull;
 import io.reactivex.disposables.Disposable;
 import io.reactivex.exceptions.Exceptions;
 import io.reactivex.functions.Function;
-import io.reactivex.internal.disposables.*;
-import io.reactivex.internal.schedulers.*;
+import io.reactivex.internal.disposables.EmptyDisposable;
```

Comment:
Please make sure your IDE doesn't unroll star imports to reduce the diff size.

Reply:
ok